draft-ietf-anima-brski-prm.md

```diff
@@ -520,127 +540,146 @@ In {{!RFC8995}}, pledges instead need to continuously request enrollment from a
 The benefits of BRSKI-PRM can be achieved even without the operational complexity of standalone Registrar-Agents by integrating the necessary functionality of the Registrar-Agent as a module into the domain registrar as shown in {{uc4figure}} so that it can support the BRSKI-PRM communications to the pledge.
 
 
+
 ## Agent-Proximity Assertion {#agt_prx}
 
-"Agent-proximity" is a statement in the PVR and in the voucher, that the registrar certificate was provided via the Registrar-Agent as defined in {{exchanges_uc2}} and not directly to the pledge.
-Agent-proximity is therefore a different assertion than "proximity", which is defined in {{Section 4 of RFC8366}}.
-Agent-proximity is defined as additional assertion type in {{I-D.ietf-anima-rfc8366bis}}.
-This assertion can be verified by the registrar and also by the MASA during the voucher-request processing.
+"Agent-proximity" is a statement in the PVR and in Voucher that the registrar LDevID certificate was provided via the Registrar-Agent as defined in {{exchanges}} and not directly to the pledge.
+It is therefore a different assertion than "proximity", which is defined in {{Section 3 of !RFC8995}}. TODO(check correct RFC; was RFC8366)
```

Comment:
My best guess is that RFC 8995 is meant -- there is another, previous comment on this.

Reply:
Yes, as above, Section 3 of RFC 8995 is the right one.